src/content/docs/en/extend/add-view-transitions.mdx

```diff
@@ -0,0 +1,434 @@
+
+    📚 Read more about the different [transition directives](/en/guides/view-transitions/#transition-directives) and [customizing animations](/en/guides/view-transitions/#customizing-animations).
+
+### Force a full browser reload for some links
+
+9. Prevent client-side routing and instead require the browser to reload when navigating to your About page.
+
+    Sometimes you will want a full browser reload when visitors click a certain link. For example, you may be linking to a page that does not also use the `<ViewTransitions />` router, or to a file directly such as a `.pdf`. 
+
+    To make it so that your browser refreshes every time you click the navigation link to go to your About page, add the `data-astro-reload` attribute to the `<a>` tag in your `<Navigation/>` component. This will override the `<ViewTransitions />` router entirely, and any of the view transition animations, for this one link.
+
+     ```astro title="src/components/Navigation.astro" ins='data-astro-reload'
+      ---
+      ---
+      <div transition:animate="slide" class="nav-links">
+        <a href="/">Home</a>
+        <a href="/about/" data-astro-reload >About</a>
+        <a href="/blog/">Blog</a>
+        <a href="/tags/" >Tags</a>
+      </div>
+      ```
+    
+    Now, when you click the navigation link to your About page, no animations will occur. The page links and title will not slide in, and the page content will not fade in when you navigate to your About page **using this link**.
+
+10. Add a link to your About page from your author name in your Markdown layout for blog posts   
+
+    `data-astro-reload` only triggers a full browser refresh when going to a new page **from the link it is added to**. It does not control all instances of navigating to your About page.
+
+    In your `<MarkdownPostLayout />` component, add a link to your About page on your author name:
+
+    ```astro title="src/layouts/MarkdownPostLayout.astro" ins='<a href="/about/">' ins="</a>"
+    ---
+    import BaseLayout from "./BaseLayout.astro";
+    import { fade } from 'astro:transitions';
+    const { frontmatter } = Astro.props;
+    ---
+
+    <BaseLayout pageTitle={frontmatter.title}>
+      <p>{frontmatter.pubDate.slice(0, 10)}</p>
+      <p transition:animate={fade({ duration: '2s' })} ><em>{frontmatter.description}</em></p>
+      <p>Written by: <a href="/about/">{frontmatter.author}</a></p>
+      <img src={frontmatter.image.url} width="300" alt={frontmatter.image.alt} />
+      <slot />
+    </BaseLayout>
+    ```
+
+    If you visit any blog post in your browser preview, and then click on the linked author name to be taken to the About page, what does the page navigation look like?
+
+    <p>
+      When a visitor clicks a link to the About page from an individual blog post, the page title and header navigation links<Spoiler>slide in across the screen</Spoiler> because <Spoiler>the `data-astro-reload` attribute is not set on these links</Spoiler>
```

Comment:
```suggestion
      When a visitor clicks a link to the About page from an individual blog post, the page title and header navigation links <Spoiler>slide in across the screen</Spoiler> because <Spoiler>the `data-astro-reload` attribute is not set on these links</Spoiler>
```